global_vars.tf

```diff
@@ -10,12 +10,82 @@ variable "aws_access_key" {
   description = "Amazon Web Service Access Key"
 }
 
-variable "submitted_responses_read_capacity" {
-  description = "The number of Read units for the Submitted Responses table"
+variable "submitted_responses_min_read_capacity" {
+  description = "The min number of Read units for the Submitted Responses table"
   default     = 25
 }
 
-variable "submitted_responses_write_capacity" {
-  description = "The number of Write units for the Submitted Responses table"
+variable "submitted_responses_max_read_capacity" {
+  description = "The max number of Read units for the Submitted Responses table"
+  default     = 500
```

Comment:
I would set these a lot higher as they are production default

Reply:
5000?
default account limit is 20,000 read 20,000 write so will give even split across tables.
we can tune and request more once its in